data_analysis/main.py

```diff
@@ -0,0 +1,65 @@
+# This file is best run as an interactive Jupyter notebook
+
+# %% import libraries
+import pandas as pd
+
+# %% read food data
+food = pd.read_csv('../data/fdc/food.csv').set_index('fdc_id')
```

Comment:
also might be worth writing the directory as a separate variable s.t. all other users know where to point the data (or at least know to change it)

Reply:
Added in https://github.com/googleinterns/amaranth/pull/1/commits/3e909e93b9fa8643c9f4a34c15fd62cf242114a5